components/cart/cart-items.tsx

```diff
@@ -13,71 +16,87 @@ export const CartItems = () => {
 
     return (
         <ul>
-            {cart.items.map((item: CartItem) => (
-                <li key={`${item.variant.sku}`} className="flex mb-4 justify-between">
-                    <div className="flex w-full">
-                        <div className="shrink-0 relative h-24 w-20 aspect-square border border-muted rounded-sm overflow-hidden">
-                            <Image {...item.images[0]} className="object-cover" />
-                        </div>
-                        <div className="flex flex-col pl-4 text-dark w-full justify-between">
-                            <div className="flex flex-col">
-                                <span className="text-lg font-bold">{item.name}</span>
-                                <span className="text-sm italic text-dark/70">{item.variant.sku}</span>
+            {cart.items.map((item: CartItem, index) => (
+                <Fragment key={`${item.variant.sku}`}>
```

Comment:
Here `${item.variant.sku}` - I guess this is string literal because the sku could be undefined, is in't it better in this case to use the index as a key?

Reply:
Good catch, I wanted to add something else here, but I guess it's not necessary since SKUs are supposed to be unique.